### DISCUSSION TITLE
[Feature] More Languages Support

### DISCUSSION THREAD

**Discussion by feikong233** [bridge-core/bridge.#283] [Ideas]
**Summary**
What should I do if I want to translate this project into another language like Chinese ?

**Alternatives/Considerations**
This really a great software and help me a lot.But English isn’t so convenient to people who are not born to speak English.So can I translate this software  into another language in the form of plug-ins?
(Just like Chinese support in VS Code.)

**Comment by solvedDev** [comment]
We've had some discussions around this and decided that we'll add support for multiple languages :)

**Comment by feikong233** [comment]
Thanks!
